starter-code/public/Ironteam.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+const User           = require("/home/manuel/ironhack/semaine5/jour1/lab-passport-roles/starter-code/model/user");
+
+var Gonzalo = User({
+  name: 'Gonzalo',
+  password: 'Gonzalo',
+  role: 'BOSS'
+});
+
+var Lluis = User({
+  name: 'Lluis',
+  password: 'Lluis',
+  role: 'DEVELOPPER'
+});
+
+var Alfonso = User({
+  name: 'Alfonso',
+  password: 'Alfonso',
+  role: 'TA'
+});
+
+
```

Comment:
Para hacerlo todo en un mismo paso puedes meter los 3 usuarios en un array, por ejemplo let users = [Lluis, Alfonso, Gonzalo] y luego haces User.create(users, (err, docs) =>{
  if (err) {throw err}
  console.log('Users created in database');
  mongoose.connection.close();
})